Makefile

```
@@ -36,7 +40,7 @@ build: vendor build-css
 
 	# Latest gsctl version
 	mkdir -p build/layouts/shortcodes
-	curl -s https://downloads.giantswarm.io/gsctl/VERSION > build/layouts/shortcodes/gsctl_version.html
+	curl -s https://api.github.com/repos/giantswarm/gsctl/releases/latest|jq -r .tag_name > build/layouts/shortcodes/gsctl_version.html
```

```suggestion
	curl -s https://api.github.com/repos/giantswarm/gsctl/releases/latest | jq -r .tag_name > build/layouts/shortcodes/gsctl_version.html
```
without space did not work for me